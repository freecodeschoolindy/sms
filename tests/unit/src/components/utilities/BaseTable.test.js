import BaseTable from '@/components/utilities/BaseTable.vue';

import testHelpers from '@@/testHelpers.js';

describe('BaseTable.vue', () => {
  let localVue;
  let store;
  let setup;

  let permissions = ['student'];

  beforeEach(() => {
    setup = testHelpers.setupTestVariables(permissions);
    localVue = setup.localVue;
    store = setup.store;
  });

  test('Renders correctly', async () => {
    const wrapper = await testHelpers.shallowMountWrapper(BaseTable, {
      localVue,
      store
    });

    expect(wrapper)
      .toMatchSnapshot();
  });
});
