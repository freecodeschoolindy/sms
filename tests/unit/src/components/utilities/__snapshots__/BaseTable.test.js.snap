// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseTable.vue Renders correctly 1`] = `
<div>
  <vue-good-table-stub theme mode="local" styleclass="vgt-table bordered" columns rows responsive="true" groupoptions="[object Object]" selectoptions="[object Object]" sortoptions="[object Object]" paginationoptions="[object Object]" searchoptions="[object Object]"></vue-good-table-stub>
</div>
`;
